cert/CACreationScript.sh

```diff
@@ -119,7 +139,13 @@ openssl ca -create_serial -cert $ROOT.crt -keyfile $ROOT.pkey -outdir $CSR -batc
 CBSD_EE=cbsd_req
 CBSD_EE_SIGN=$CBSD_EE$SIG
 
-openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=CBSD End-Entity Example" -config openssl.cnf
+if [ -f $CBSD_EE.pkey ]
+then
+  openssl req -new -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -key $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=CBSD End-Entity Example" -config openssl.cnf
```

Comment:
If you're generating the CSR as an example, why not just always generate it? If it needs the cbsd_req.pkey just do so after that gets created.

Reply:
If user has provide CBSD device end entity key already while running bash script, this conditional statement is added to avoid RSA key generation for CBSD device.
So script shall only generate CSR for the case.